.github/workflows/ci-lint.yml

```diff
@@ -0,0 +1,29 @@
+# NOTE: This name appears in GitHub's Checks API and in workflow's status badge.
+name: ci-lint
+
+# Trigger the workflow when:
+on:
+  # A push occurs to one of the matched branches.
+  push:
+    branches: [master]
+  # Or when a pull request event occurs for a pull request against one of the
+  # matched branches.
+  pull_request:
+    branches: [master]
+
+jobs:
+  lint:
+    # NOTE: This name appears in GitHub's Checks API.
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Set up Node.js 18
+        uses: actions/setup-node@v3.5.0
+        with:
+          node-version: '18.x'
+          cache: yarn
+      - run: yarn install --frozen-lockfile | tee yarn-install-out
```

Comment:
Can we add `name` field to all the steps that explains what the step does?

Same for other workflows.